stock_picking_group_by_partner_by_carrier/report/report_delivery_slip.xml

```diff
@@ -12,7 +12,7 @@
             expr='//table[@name="stock_move_table"]/tbody/tr/td[span[@t-field="move.product_id"]]'
             position="replace"
         >
-            <t t-if="move.product_id">
+            <t t-if="not move.is_fake">
```

Comment:
You can replace the `is_fake` field by `not move.id`:
"new" records have a `NewID` instance is id, which is evaluated as False: https://github.com/odoo/odoo/blob/b83c48d5c48954193227bb1d5cfa45f8feb92d60/odoo/models.py#L195-L196

Reply:
Ok got it.
Removed the is_fake fields